doc/cephfs/index.rst

```diff
@@ -1,33 +1,38 @@
 .. _ceph-filesystem:
 
 =================
- Ceph Filesystem
+ Ceph File System
 =================
 
-The Ceph Filesystem (CephFS) is a POSIX-compliant filesystem that uses
-a Ceph Storage Cluster to store its data. The Ceph filesystem uses the same Ceph
-Storage Cluster system as Ceph Block Devices, Ceph Object Storage with its S3
-and Swift APIs, or native bindings (librados).
+The Ceph Filesystem, or **CephFS**, is a POSIX-compliant file system built on
+top of Ceph's distributed object store, RADOS. CephFS endeavors to provide a
+state-of-the-art, multi-use, and performant file store for a variety of
+applications, including traditional use-cases like shared home directories, HPC
+scratch space, and distributed workflow shared storage.
+
+CephFS achieves these goals through the use of some novel architectural
+choices.  Notably, file metadata is separated from file data and served via
+cluster of *Metadata Servers*, or **MDS**. Clients of the file system also have
+direct access to RADOS for reading and writing file data blocks. This allows
+workloads to scale with with the size of the underlying RADOS object store.
+Data access is coordinated through the cluster of MDS which serve as
+authorities for the state of the distributed metadata cache maintained by both
+clients and MDS. Mutations to metadata are aggregated by each MDS into a series
+of efficient writes to a journal on RADOS; no metadata state is stored locally
+by the MDS. This architecture is visually depicted in the below diagram.
```

Comment:
I'd drop this last sentence. Seems superfluous.

Reply:
done